packages/teams-js/src/public/dialog.ts

```diff
@@ -21,7 +21,15 @@ import {
 import { runtime } from './runtime';
 
 /**
- * Namespace to interact with the dialog module-specific part of the SDK.
+ * This group of capabilities enables apps to show dialogs. There are two primary types of dialogs: URL-based dialogs and [Adaptive Card](https://learn.microsoft.com/adaptive-cards/) dialogs.
+ * Both types of dialogs are shown on top of your app, preventing interaction with your app while they are displayed.
+ * - URL-based dialogs allow you to specify a URL for which the contents will be shown inside the dialog.
+ *   - For URL dialogs, you should use the functions and interfaces in the {@link dialog.url} namespace.
```

Comment:
For URL dialogs, **use the** functions and interfaces . . . 

(Drop "you should" to be more concise, per [MS Docs styleguide](https://review.learn.microsoft.com/en-us/help/contribute/writing-principles?branch=main#3-write-concisely))